pontoon/base/static/js/lib/fluent-syntax.js

```diff
@@ -1,15 +1,31 @@
-/* fluent-syntax@0.3.0 */
+/* fluent-syntax@0.4.0 */
```

Comment:
Thanks for the pointer! `FluentSyntax.parse()` will create a new instance of the parser on every call. I'd recommend instantiating the parser once and using `parseEntry` instead of `.body[0]`:

```javscript
var parser = FluentSyntax.FluentParser({ withSpans: false });
var entity_ast = parser.parseEntry(entity.original);
```